docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to Phonebook 
+
+## How can I contribute?
+### Reporting Bugs
```

Comment:
add a blank line between markdown elements. You can use https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack to support your writing